### PR TITLE
Adding documentation on "shared with users" default filter in work pa…

### PR DESCRIPTION
**Pull request by MayaBerd**
…ckages

Modified pages:
User Guide -> Work Packages -> Share work packages User Guide -> Work Packages -> Work package views

### CODE REVIEW

File: docs/user-guide/work-packages/share-work-packages/README.md [hunk 1]
```diff
@@ -60,4 +62,12 @@ You can always adjust the viewing rights of a user by selecting an option from t
 
 ![](openproject_user_guide_shared_with_list_change_role.png)
 
+## Remove sharing privileges
+
 You can also remove the user from the list by clicking on **Remove** next to the user name. Please not that this will not remove a user entirely, but only revoke the viewing and/or editing rights for the work package. User account will remain intact. If you need to [delete a user](../../../system-admin-guide/users-permissions/users/#delete-users), please do that in system administration or contact your administrator.
```
Comment (marcalcobe) [inline docs/user-guide/work-packages/share-work-packages/README.md]: There is a small mistake on the second sentence "Please note that..."

File: docs/user-guide/work-packages/share-work-packages/README.md [hunk 2]
```diff
@@ -60,4 +62,12 @@ You can always adjust the viewing rights of a user by selecting an option from t
...
+
+## Shared work packages overview
+
+For an overview of all work packages that have been shared within a specific project navigate to that project, select the module **Work Packages** and choose the filter **Shared with users** from the list of default work package filters on the left side. You can then decide if you or a project administrator needs to change or remove sharing options. You can also [adjust this filter](./work-package-table-configuration/#filter-work-packages) and save it under your private work package filters.
```
Comment (marcalcobe) [inline docs/user-guide/work-packages/share-work-packages/README.md]: Hi Maya, I think this menu entry also is present in a global level. Therefore I don't know if we should only talk about the specific project level.

Comment (marcalcobe) [reply docs/user-guide/work-packages/share-work-packages/README.md]: I find this sentence a bit confusing: "You can then decide if you or a project administrator needs to change or remove sharing options." as not only project admins can do changes, any role with the proper permission will be able to. 

Maybe I would change it to "You, with the correct permissions, can decide to change or remove sharing options."

**Review by marcalcobe — CHANGES_REQUESTED** [review]
Hi @MayaBerd, I've added my review.

File: docs/user-guide/work-packages/share-work-packages/README.md (hunk 2 above)
Comment (MayaBerd) [reply docs/user-guide/work-packages/share-work-packages/README.md]: @marcalcobe thanks, a good point. I've added the part on the global level and also a sentence about the specific level case.

**Pull request merged**